pkg/cmd/printer/printer.go

```diff
@@ -243,9 +240,8 @@ func (p tableEventPrinter) Print(event trace.Event) {
 		switch p.containerMode {
 		case ContainerModeDisabled:
 			fmt.Fprintf(p.out,
-				"%-16s %016x  %-16s %-13s %-12d %-12d %-6d %-16s %-7d %-7d %-7d %-16d %-25s ",
+				"%-16s %-17s %-13s %-12d %-12d %-6d %-16s %-7d %-7d %-7d %-16d %-25s ",
```

Comment:
Tested and it's ok. The removed column had no space padding in the format (since it was x), so we need to insert a space between those columns. When we remove it, the next field gets one more space, so 17.

```shell
❯ sudo ./dist/tracee -p examples/new_containers.yaml -o table-verbose
Place your finger on the fingerprint reader
{"level":"error","ts":1680016138.0945082,"msg":"Opening plugin /home/gg/code/tracee/dist/signatures/builtin.so: plugin.Open(\"/home/gg/code/tracee/dist/signatures/builtin\"): plugin was built with a different version of package github.com/aquasecurity/tracee/types/protocol"}
{"level":"error","ts":1680016138.0945618,"msg":"Requested capabilities","error":"plugin.Open(\"/home/gg/code/tracee/dist/signatures/builtin\"): plugin was built with a different version of package github.com/aquasecurity/tracee/types/protocol"}
TIME             UTS_NAME          CONTAINER_ID  MNT_NS       PID_NS       UID    COMM             PID/host        TID/host        PPID/host       RET              EVENT                     ARGS
12:09:12:634733  hb                              4026531841   4026531836   0      systemd          1      /1       1      /1       0      /0       0                container_create          runtime: docker, container_id: 9978cd7115bbc82d7c677206b06681533251312ef4634499662bea61d877cc43, ctime: 1680016152635525632, container_image: , container_image_digest: , container_name: , pod_name: , pod_namespace: , pod_uid: , pod_sandbox: false
12:09:12:806650  9978cd7115bb      9978cd7115bb  4026534705   4026534708   0      sh               1      /129527  1      /129527  129505 /129505  0                security_file_open        pathname: /dev/tty, flags: O_RDWR|O_LARGEFILE, dev: 66, inode: 8, ctime: 1680016152668192698, syscall_pathname: /dev/tty
12:09:12:806700  9978cd7115bb      9978cd7115bb  4026534705   4026534708   0      sh               1      /129527  1      /129527  129505 /129505  0                security_file_open        pathname: /etc/passwd, flags: O_RDONLY|O_LARGEFILE, dev: 61, inode: 3408099, ctime: 1679404243493046884, syscall_pathname: /etc/passwd
12:09:12:806703  9978cd7115bb      9978cd7115bb  4026534705   4026534708   0      sh               1      /129527  1      /129527  129505 /129505  0                security_file_open        pathname: /etc/passwd, flags: O_RDONLY|O_LARGEFILE|O_NOATIME, dev: 271581188, inode: 3408099, ctime: 1679404243493046884, syscall_pathname: /etc/passwd
```